website/notifications/constants.py

```diff
@@ -4,7 +4,9 @@
 }
 
 USER_SUBSCRIPTIONS_AVAILABLE = {
-    'comment_replies': 'Replies to your comments'
+    'global_comment_replies': 'Replies to your comments',
```

Comment:
This reminds me: We should have a field validator for `NotificationSubscription.event_name` and `NotificationDigest.event` that verifies that the value is one of the keys in `USER_SUBSCRIPTIONS_AVAILABLE` or `NODE_SUBSCRIPTIONS_AVAILABLE`.

But that's out of the scope of this feature...I'll create a ticket.